src/Symfony/Component/EventDispatcher/composer.json

```diff
@@ -17,8 +17,8 @@
     ],
     "require": {
         "php": ">=7.2.5",
-        "symfony/deprecation-contracts": "^2.1",
-        "symfony/event-dispatcher-contracts": "^2",
+        "symfony/deprecation-contracts": "^2.1|^3",
+        "symfony/event-dispatcher-contracts": "^2|^3",
```

Comment:
Is this one safe ? EventDispatcher implements the contracts interface, it does not only consume it.

Reply:
Despite the major version bump, v3 does not contain any breaking changes. If your code is compatible with v2, it's also compatible with v3.

https://github.com/symfony/event-dispatcher-contracts/compare/2.5...main

Reply:
I forgot that v2 already has the return type